modROS.lua

```diff
@@ -336,8 +332,8 @@ function ModROS:publish_laser_scan_func()
         -- serialise to JSON and write to pipe
         self.file_pipe:write(sensor_msgs_LaserScan.ros_msg_name .. "\n" .. scan_msg:to_json())
 
-        -- calculate the angle of the current layer's scanning plane wrt origin of the laser
-        local scanning_plane_pitch = -math.atan(delta_y * i / radius)
+        -- the angle of the current layer's scanning plane wrt origin of the laser
+        local scanning_plane_pitch = 0
         -- convert to quaternion for ROS TF; scanning_plane_pitch is the angle rotated from the origin of the laser scanner along y axis.
         local q = ros_quaternion.from_euler(0, scanning_plane_pitch, 0)
```

Comment:
Seeing as there is no inclination any more of the planes, I'd suggest removing this constant.

```suggestion
        -- convert to quaternion for ROS TF
        local q = ros_quaternion.from_euler(0, 0, 0)
```

something to consider: do we even need to convert anything here?

Could just set a `[0, 0, 0, 1]` quaternion later in `tf_base_link_laser_frame_i`.

Reply:
we don't actually. I thought it would be nice to keep it but agreed, it's better to just remove it.